### ISSUE TITLE
Installation failed: cp: /etc/bash_completion.d/sysget: No such file or directory

### ISSUE THREAD

**Issue by robrecord** [emilengler/sysget#39]
When I install I get this: `cp: /etc/bash_completion.d/sysget: No such file or directory`

MacOS 10.15.1
Homebrew installed

Here is the full output.

```bash
~/Repositories
❯ git clone git@github.com:emilengler/sysget.git
Cloning into 'sysget'...
remote: Enumerating objects: 917, done.
remote: Total 917 (delta 0), reused 0 (delta 0), pack-reused 917
Receiving objects: 100% (917/917), 375.61 KiB | 1.18 MiB/s, done.
Resolving deltas: 100% (583/583), done.

~/Repositories took 2s
❯ cd sysget

sysget on  master
❯ make && sudo make install
g++ -std=c++11 -Wall -Wextra -Wpedantic -c src/main.cpp src/main.hpp
g++ -std=c++11 -Wall -Wextra -Wpedantic -c src/packagemanager.hpp src/packagemanager.cpp
g++ -std=c++11 -Wall -Wextra -Wpedantic -c src/utils.hpp src/utils.cpp
g++ -std=c++11 -Wall -Wextra -Wpedantic -c src/translation.hpp src/translation.cpp
g++ -std=c++11 -Wall -Wextra -Wpedantic -o sysget main.o packagemanager.o utils.o translation.o
Password:
cp sysget /usr/local/bin/sysget
mkdir -p /usr/local/man/man8
cp contrib/man/sysget.8 /usr/local/man/man8/sysget.8
gzip /usr/local/man/man8/sysget.8
cp contrib/sysget.bash-completion /etc/bash_completion.d/sysget
cp: /etc/bash_completion.d/sysget: No such file or directory
make: *** [install] Error 1
```

**Comment by robrecord** [comment]
I found the directory `/etc/bash_completion.d` is missing in my system.

I created it with `sudo mkdir` and now the installation works.

Could this be something the installer handles?

**Comment by cvengler** [comment]
@robrecord I fixed that in 023e230.
However that fix is a bit dirty. When I initially made sysget, I wasn't really experienced with build systems. I will soon port sysget to autoconf or Cmake.
